Community/Tdarr_Plugin_DOOM_NVENC_Tiered_MKV_CleanAll.js

```diff
@@ -57,6 +57,50 @@ const details = () => {
                 \\nExample 14 Mbps:\\n
                 14000000`,
       },
+      {
+        name: "target_cq_480p576p",
+        type: 'number',
+        defaultValue: '29',
+        inputUI: {
+          type: 'text',
+        },
+        tooltip: `Specify the target constant quality for 480p and 576p files. Lower value equals less compression.
+                \\nDefault:\\n
+                29`,
+      },
```

Comment:
You have to change the defaults from a string to a number too :) so remove the single quotes around it. Goes for all of them